Cargo.toml

```diff
@@ -34,3 +34,6 @@ serde_json = { version = "1.0", optional = true }
 [dev-dependencies]
 simple_logger = "1.0.1"
 matches = "0.1"
+
+[package.metadata.release]
+disable-publish = true
```

Comment:
Please add a comment as to why this is needed. I initially thought this was for cargo-release, but the docs below recommend using the `--skip-publish` parameter.

Reply:
It is for cargo release. This is a fallback in case we forget to use the flag.

Reply:
Thanks for dealing with this in https://github.com/ballista-compute/sqlparser-rs/pull/263 !